hack/make-rules/test-integration.sh

```diff
@@ -94,7 +95,7 @@ checkEtcdOnPath
 trap cleanup EXIT
 
 # Convert the CSV to an array of API versions to test
-IFS=';' read -a apiVersions <<< "${KUBE_TEST_API_VERSIONS}"
+IFS=';' read -r apiVersions <<< "${KUBE_TEST_API_VERSIONS}"
```

Comment:
Why are you doing this? What shellcheck failure are you fixing by doing this?

Reply:
Why did I modify and shellcheck like this lead to what failure：
```
In ./hack/make-rules/test-integration.sh line 98:
IFS=';' read -a apiVersions <<< "${KUBE_TEST_API_VERSIONS}"
        ^--^ SC2162: read without -r will mangle backslashes.

For more information:
  https://www.shellcheck.net/wiki/SC2162 -- read without -r will mangle backs...
```

Reply:
should be `read -a` -> `read -ar` then, a is a different flag. https://ss64.com/bash/read.html